.eslintrc.js

```diff
@@ -73,8 +73,8 @@ module.exports = {
 		// es6 import/export and require
 		'node/no-unpublished-require': ['off'],
 		'node/no-unsupported-features/es-syntax': ['off'],
-		// kebab case components for vuejs
-		'vue/component-name-in-template-casing': ['error', 'kebab-case'],
+		// check case of component names
```

Comment:
:(

Reply:
https://github.com/nextcloud/social/blob/master/.eslintrc.js#L54
https://github.com/nextcloud/contacts/blob/master/.eslintrc.js#L77
https://github.com/nextcloud/firstrunwizard/blob/master/src/App.vue#L2

@nextcloud/vue we should probably decide a unified eslint for all of our apps.
Unity is better. Same goes for our coding style :)

Reply:
The background for this change was this strong recommendation: https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended

But yes, in general we should use the same standards. But we should also orient on the the official vue style guide.

Reply:
I'm actually fine with using Pascal as well :)
Iirc we went for this config on the first app (contacts) that started using eslint but I think he recommendation were not those. Because I remember starting with Pascal until we read somewhere that kebab was recommended by vue. Now it's the opposite :thinking: 
I'm confused, but if the guidelines says so, let's migrate to pascale? :wink:

Reply:
Maybe, they changed their mind in the meanwhile. Also, it's not easy to understand the different recommendations for different template types (single-file vs. string vs. DOM).

But yeah, let's use PascalCase, here. Maybe we can publish a `.eslintrc.js` master file somewhere in a repository, where we can discuss common conventions in the future. E.g., I really like `comma-dangle: always-multiline`, but I think most NC projects don't use this.  :wink: